test/integration/auth-methods/edgegrid.test.js

```diff
@@ -0,0 +1,399 @@
+
+            expect(request.headers.get('authorization')).to.be.undefined;
+        });
+
+        it('should have completed the run', function () {
+            expect(testrun).to.be.ok;
+            expect(testrun).to.nested.include({
+                'done.callCount': 1
+            });
+            testrun.done.getCall(0).args[0] && console.error(testrun.done.getCall(0).args[0].stack);
+            expect(testrun.done.getCall(0).args[0]).to.be.null;
+            expect(testrun).to.nested.include({
+                'start.callCount': 1
+            });
+        });
+
+        it('should have sent the request once', function () {
+            expect(testrun).to.nested.include({
+                'request.callCount': 1
+            });
+
+            var request = testrun.request.getCall(0).args[3],
+                response = testrun.request.getCall(0).args[2];
+
+            expect(request.url.toString()).to.eql(edgeGridAuthServer.url);
+            expect(response).to.have.property('code').that.is.oneOf([200, 401]);
+        });
+
+        it('should not have passed the EdgeGrid authorization', function () {
+            expect(testrun).to.nested.include({
+                'request.callCount': 1
+            });
+
+            var request = testrun.request.getCall(0).args[3],
+                response = testrun.request.getCall(0).args[2];
+
+            expect(request.url.toString()).to.eql(edgeGridAuthServer.url);
+            expect(response).to.have.property('code', 401);
+        });
+    });
+
+    describe('with correct credentials', function () {
+        var testrun;
+
+        before(function (done) {
+            // perform the collection run
+            this.run({
+                collection: {
+                    item: {
+                        request: {
+                            auth: {
+                                type: 'edgegrid',
+                                edgegrid: credentials
+                            },
+                            url: edgeGridAuthServer.url,
+                            method: 'GET'
+                        }
+                    }
+                }
+            }, function (err, results) {
+                testrun = results;
+                done(err);
+            });
+        });
+
+        it('should include correct Authorization header', function () {
+            var request = testrun.request.getCall(0).args[3],
+                header = request.headers.members[0],
+                nonceRegex = /nonce=([A-Z]|[a-z]|[0-9]|-){36};/,
+                timestampRegex = /timestamp=[0-9]{8}T[0-9]{2}:[0-9]{2}:[0-9]{2}\+0000;/;
+
+            expect(header).to.have.have.property('key', 'Authorization');
+            expect(header).to.have.have.property('value').that.include(`client_token=${credentials.clientToken}`);
+            expect(header).to.have.have.property('value').that.include(`access_token=${credentials.accessToken}`);
```

Comment:
split these into multiple lines.
```suggestion
            expect(header).to.have.have.property('value')
              .that.include(`access_token=${credentials.accessToken}`);
```